.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "external/microprofile"]
```

Comment:
This will certainly be useful!
I would put external dependencies in a sub-directory of `external`.
`librw` will also become a dependency and can then be put alongside `microprofiler`.

I propose the following directory structure:
```
external
├── CMakeLists.txt
├── microprofiler
│    ├── microprofiler/  # git submodule
│    └── CMakeLists.txt
├── librw
│    ├── librw/          # git submodule
│    └── CMakeLists.txt
└── librwgta
     ├── librwgta/       # git submodule
     └── CMakeLists.txt
       
```
I did something similar here: https://github.com/madebr/openrw/tree/librw/external